src/foam/u2/crunch/CapabilityCardView.js

```diff
@@ -74,6 +74,11 @@ foam.CLASS({
             var statusEnum =  foam.nanos.crunch.CapabilityJunctionStatus.AVAILABLE;
             if ( ucj ) {
               statusEnum = ucj.status;
+              
+              // Want approved to be displayed same as pending because this status denotes that
+              // the ucj has passed review, but its prerequites are still to be reviewed and in pending status
+              if ( statusEnum === foam.nanos.crunch.CapabilityJunctionStatus.APPROVED ) 
```

Comment:
So wait, will we ever have an APPROVED status UCJ appear through the CapabilityCardView?

Reply:
no, users should only see available, action_required, pending, and granted

Reply:
and expired